packages/liveblocks-react/src/room.tsx

```diff
@@ -100,10 +100,13 @@ import type {
   ThreadsState,
   ThreadsStateSuccess,
   ThreadSubscription,
+  UseStorageStatusOptions,
   UseThreadsOptions,
 } from "./types";
 import { useScrollToCommentOnLoadEffect } from "./use-scroll-to-comment-on-load-effect";
 
+const SMOOTH_DELAY = 1000;
```

Comment:
Currently, the `"synchronized"` event will be delivered after 1 second. If you want to see it live, run the `e2e/next-sandbox` app (`turbo run dev`) and open [this page](http://localhost:3007/storage/list?room=e2e-storage-list).


https://github.com/liveblocks/liveblocks/assets/83844/7d770d5d-7843-40a7-a3c5-4722d4940c9f